frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -190,6 +195,20 @@ class Slider extends React.PureComponent<Props, State> {
     return sprintf(format, value)
   }
 
+  private thumbValueAlignment(): void {
+    const slider = this.sliderRef.current
+    const thumb = this.thumbValueRef.current
+
+    if (slider && thumb) {
+      const sliderPosition = slider.getBoundingClientRect()
+      const thumbPosition = thumb.getBoundingClientRect()
+
+      thumb.style.left = thumbPosition.left < sliderPosition.left ? "0px" : ""
+      thumb.style.right =
+        thumbPosition.right > sliderPosition.right ? "0px" : ""
+    }
+  }
+
```

Comment:
Open to other suggestions, but `.getBoundingClientRect()` seems to be the simplest solution to cover different slider input types and test whether the thumbValue is outside the slider's bounds.

Reply:
`getBoundingClientRect()` is totally fine